domains/mraxays.json

```
@@ -4,6 +4,7 @@
     "discord": "mrakshayas"
   },
   "records": {
-    "A": ["216.150.1.1"]
+    "A": ["216.150.1.1"],
+    "TXT": ["peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"]
```

its just a single text and for readability
```suggestion
    "TXT": "peerlist-domain-verification=uhp69nnnlqmkdd9hmrlemm6keqnd"
```